src/openrct2/rct2/S6Exporter.cpp

```diff
@@ -1039,10 +1039,9 @@ void S6Exporter::ExportSpriteVehicle(RCT2SpriteVehicle* dst, const Vehicle* src)
     else
     {
         uint16_t trackType = src->GetTrackType();
-        int8_t trackDirection = src->GetTrackDirection();
         if (trackType == TrackElemType::Booster)
             trackType = TrackElemType::BoosterAlias;
-        dst->track_type = (trackType << 2) | (trackDirection & 3);
+        dst->track_type = (trackType << 2) | (src->track_direction & 3);
```

Comment:
Why?

Reply:
This is correct. vehicle->track_type contains the track_direction as well. It might be an idea to keep the comment:
```
 // Track direction and type are in the same field
```

Reply:
I'd much rather have a function for this.